encoding/v2/kzg/prover/backend/gnark/multiframe_proof.go

```diff
@@ -47,30 +47,29 @@ func NewMultiProofBackend(
 // 2. https://eprint.iacr.org/2023/033.pdf (how to compute the single multiproof fast)
 // 3. https://github.com/khovratovich/Kate/blob/master/Kate_amortized.pdf (fast multiple multiproofs)
 func (p *KzgMultiProofBackend) ComputeMultiFrameProofV2(
-	polyFr []fr.Element, numChunks, chunkLen, numWorker uint64,
+	polyFr []fr.Element, numTotalChunks, toeplitzMatrixLen, chunkLen, numWorker uint64,
```

Comment:
you mention in the PR description that `toeplitzMatrixLen = blobLen / chunkLen`.
So iiuc you don't need to take toeplitzMatrixLen as an argument here since you can just define it as
```
toeplitzMatrixLen := len(polyFr) / chunkLen
```